docs/how-to/index.md

```diff
@@ -10,4 +10,5 @@ deploy_sdcore_cups
 deploy_sdcore_gnbsim
 integrate_sdcore_with_observability
 enable_hugepages_for_upf
+run_upf_in_dpdk_mode
```

Comment:
```suggestion
deploy_sdcore_user_plane_in_dpdk_mode
```

Reply:
And I'd move it next to the other deploys

Reply:
Done.